releasenotes.txt

```diff
@@ -0,0 +1,315 @@
+BISDN Linux
+4.8.0 Release Notes
+Date: 2022-09-05
+
+! New in this release
+- added Edgecore EPS201 (AS4630-54TE) as supported platform
+- updated to Linux kernel 5.15.64
+- bundle MSTPD state as well in collect-debug-info
+- allow ignoring failing commands in bundle-debug-info
+- allow arbitrary argument order for bundle-debug-info
+
+! Changed behavior
+- install 90-enp.link to /lib instead of /etc
+- enable kernel setting system time at boot
+- set default hostname based on ONIE machine name
+- the installer now checks if the platform is supported before installing
+- there is no default bond0 device created anymore
+
+! Bug fixes
+- avoid link flapping when DHCP sets MTU
```

Comment:
This only affects intel/x86 based platforms; as4610 series is unaffected

Reply:
done